Gopkg.lock

```diff
@@ -1717,6 +1717,7 @@
     "github.com/operator-framework/operator-lifecycle-manager/pkg/controller/install",
     "github.com/pborman/uuid",
     "github.com/prometheus/prometheus/util/promlint",
+    "github.com/rogpeppe/go-internal/modfile",
```

Comment:
We won't be making the switching in operator-sdk from dep?

Reply:
I'll do that in a separate PR. There's an issue with a dep of OLM's, which requires `mercury` to pull. I'd like to see if there's a workaround. In the mean time we can support `go mod` for new projects.